contracts/ibc-callbacks/schema/cw_schema/ibc-callbacks.json

```diff
@@ -325,6 +330,11 @@
           "type": "optional",
           "inner": 18
         },
+        {
+          "name": "core::option::Option<core::option::Option<cosmwasm_std::ibc::callbacks::IbcTransferCallback>>",
```

Comment:
Hm, this is now a double option. How does the codegen handle that?

Reply:
Ah well, fair, should be encodable. something like a, well, "default" type. Gonna fix that